pkg/authorization/registry/role/rest.go

```diff
@@ -0,0 +1,176 @@
+package role
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	kerrors "github.com/GoogleCloudPlatform/kubernetes/pkg/api/errors"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/apiserver"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	policyregistry "github.com/openshift/origin/pkg/authorization/registry/policy"
+)
+
+// REST implements the RESTStorage interface in terms of an Registry.
+type REST struct {
+	registry policyregistry.Registry
+}
+
+// NewREST creates a new REST for policies.
+func NewREST(registry policyregistry.Registry) apiserver.RESTStorage {
```

Comment:
Add a TODO to add Get and List later, just for completeness (and clusterAdmin sanity)